<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ROSTTBAppWindow</class>
 <widget class="QMainWindow" name="ROSTTBAppWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1643</width>
    <height>963</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ROSTTBAppWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>590</y>
      <width>1601</width>
      <height>301</height>
     </rect>
    </property>
    <property name="title">
     <string>Add New Service</string>
    </property>
    <widget class="QTimeEdit" name="starttimeEdit">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>31</y>
       <width>118</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Start Time</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="servicerefEdit">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>30</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>40</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Service Reference</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxStartSpeed">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>70</y>
       <width>60</width>
       <height>30</height>
      </rect>
     </property>
     <property name="maximum">
      <number>999</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxMaxSpeed">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>100</y>
       <width>60</width>
       <height>30</height>
      </rect>
     </property>
     <property name="maximum">
      <number>999</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxMass">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>130</y>
       <width>60</width>
       <height>30</height>
      </rect>
     </property>
     <property name="maximum">
      <number>9999</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxForce">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>160</y>
       <width>60</width>
       <height>30</height>
      </rect>
     </property>
     <property name="maximum">
      <number>9999</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Start Speed (kph)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Max Speed (kph)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Mass (te)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Max Brake Force (te)</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxPower">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>190</y>
       <width>60</width>
       <height>30</height>
      </rect>
     </property>
     <property name="maximum">
      <number>9999</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>200</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Max Power (kW)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>80</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Description</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="descEdit">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>70</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonStandard">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>30</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Standard</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonShuttleFeeder">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>30</y>
       <width>201</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Shuttle from Feeder Service</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEditFeederRef">
     <property name="geometry">
      <rect>
       <x>820</x>
       <y>60</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEditShuttleRef">
     <property name="geometry">
      <rect>
       <x>990</x>
       <y>80</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonFeeder">
     <property name="geometry">
      <rect>
       <x>990</x>
       <y>30</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::DefaultContextMenu</enum>
     </property>
     <property name="text">
      <string>Feeder for Shuttle Service</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelFeederRef">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>70</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Feeder Reference</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelShuttleRef">
     <property name="geometry">
      <rect>
       <x>990</x>
       <y>60</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Shuttle Reference</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxRepeats">
     <property name="geometry">
      <rect>
       <x>840</x>
       <y>120</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelRepeats">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>120</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Repeats</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelTerminusWaitTime">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>150</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Time Interval (mins)</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxRepeatInterval">
     <property name="geometry">
      <rect>
       <x>840</x>
       <y>150</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonShuttleStop">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>70</y>
       <width>201</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Shuttle from Stop</string>
     </property>
    </widget>
    <widget class="Line" name="line_3">
     <property name="geometry">
      <rect>
       <x>1180</x>
       <y>30</y>
       <width>16</width>
       <height>241</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonFromOther">
     <property name="geometry">
      <rect>
       <x>990</x>
       <y>130</y>
       <width>191</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Service from Other Service</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelParentRef">
     <property name="geometry">
      <rect>
       <x>990</x>
       <y>160</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Parent Service Reference</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEditParentRef">
     <property name="geometry">
      <rect>
       <x>990</x>
       <y>180</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxParentSplit">
     <property name="geometry">
      <rect>
       <x>990</x>
       <y>220</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Parent is a split service</string>
     </property>
    </widget>
    <widget class="Line" name="line_4">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>30</y>
       <width>16</width>
       <height>241</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="Line" name="line_5">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>30</y>
       <width>16</width>
       <height>241</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="Line" name="line_6">
     <property name="geometry">
      <rect>
       <x>1190</x>
       <y>30</y>
       <width>16</width>
       <height>241</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>260</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>23</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>SERVICE PROPERTIES</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>260</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>23</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>SERVICE TYPE</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>1270</x>
       <y>250</y>
       <width>261</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>23</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>SERVICE TERMINATION</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonInsert">
     <property name="geometry">
      <rect>
       <x>1210</x>
       <y>40</y>
       <width>113</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Insert</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonDelete">
     <property name="geometry">
      <rect>
       <x>1210</x>
       <y>70</y>
       <width>113</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEditEnterID1">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>140</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEditEnterID2">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>210</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>120</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Track ID Rear</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>180</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Track ID Front</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxRefIncrement">
     <property name="geometry">
      <rect>
       <x>840</x>
       <y>180</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>180</y>
       <width>140</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Reference Increment</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Current Service</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>20</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Timetable</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonROSLoc">
    <property name="geometry">
     <rect>
      <x>1460</x>
      <y>30</y>
      <width>151</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Set ROS Location</string>
    </property>
   </widget>
   <widget class="QLabel" name="ROSStatus">
    <property name="geometry">
     <rect>
      <x>1420</x>
      <y>60</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Railway Operation Simulator Location Unset</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonRoute">
    <property name="geometry">
     <rect>
      <x>1480</x>
      <y>90</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Route</string>
    </property>
   </widget>
   <widget class="QLabel" name="SelectedRoute">
    <property name="geometry">
     <rect>
      <x>1480</x>
      <y>120</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Route Map Not Specified</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonAddLocation">
    <property name="geometry">
     <rect>
      <x>1470</x>
      <y>190</y>
      <width>141</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Calling Point</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidgetService">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>381</width>
      <height>531</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidgetTimetable">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>60</y>
      <width>941</width>
      <height>531</height>
     </rect>
    </property>
   </widget>
   <widget class="QTimeEdit" name="timeEditTTBStart">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>17</y>
      <width>118</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>20</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Time:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonTTBTime">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>13</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Confirm</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelStartTime">
    <property name="geometry">
     <rect>
      <x>674</x>
      <y>19</y>
      <width>59</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>00:00</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonTTBTimeEdit">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>13</y>
      <width>71</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Edit</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1643</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_ROSTTBGen"/>
    <addaction name="actionDocumentation"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionAbout_ROSTTBGen">
   <property name="text">
    <string>About ROSTTBGen</string>
   </property>
  </action>
  <action name="actionDocumentation">
   <property name="text">
    <string>Documentation</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
